pkg/controller/migmigration/annotation.go

```diff
@@ -63,36 +65,38 @@ type ServiceAccounts map[string]map[string]bool
 // The PvAccessModeAnnotation annotation is added to PVC as needed by the velero plugin.
 // The ResticPvBackupAnnotation is added to Pods as needed by Restic.
 // The ResticPvVerifyAnnotation is added to Pods as needed by Restic.
-// The IncludedInStageBackupLabel label is added to Pods, PVs, PVCs and is referenced
-// by the velero.Backup label selector.
+// The IncludedInStageBackupLabel label is added to Pods, PVs, PVCs, ImageStreams and
+// is referenced by the velero.Backup label selector.
 // The IncludedInStageBackupLabel label is added to Namespaces to prevent the
 // velero.Backup from being empty which causes the Restore to fail.
 func (t *Task) annotateStageResources() error {
-	client, err := t.getSourceClient()
+	sourceClient, err := t.getSourceClient()
 	if err != nil {
 		return liberr.Wrap(err)
 	}
 	// Namespaces
-	err = t.labelNamespaces(client)
+	err = t.labelNamespaces(sourceClient)
 	if err != nil {
 		return liberr.Wrap(err)
 	}
 	// Pods
-	serviceAccounts, err := t.annotatePods(client)
+	serviceAccounts, err := t.annotatePods(sourceClient)
 	if err != nil {
 		return liberr.Wrap(err)
 	}
 	// PV & PVCs
-	err = t.annotatePVs(client)
+	err = t.annotatePVs(sourceClient)
 	if err != nil {
 		return liberr.Wrap(err)
 	}
 	// Service accounts used by stage pods.
-	err = t.labelServiceAccounts(client, serviceAccounts)
+	err = t.labelServiceAccounts(sourceClient, serviceAccounts)
 	if err != nil {
 		return liberr.Wrap(err)
 	}
 
+	err = t.labelImageStreams(sourceClient)
```

Comment:
Checking the returned `err` is missing here.

Reply:
good catch, resolved